### PR TITLE
11 delete comments

### PR DESCRIPTION
**Pull request by GWorsnop**

### CODE REVIEW

File: src/components/CommentDeleteButton.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      <>
+        <button
+          className="justify-self-center text-center w-1/3 rounded-lg btn-blue"
+          onClick={handleDelete}
+          disabled={disable}
+        >
+          Delete
+        </button>
+        {error ? error : null}
+      </>
```
Comment (corkesw) [inline src/components/CommentDeleteButton.jsx]: When we get an error, we lose the button and have to refresh to get it back. Maybe a different piece of UI to denote that there is an error to save the user having this extra step.

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -73,6 +75,13 @@ function Comments({ review_id }) {
                 comment_id={comment.comment_id}
                 originalVote={comment.votes}
               />
+              {comment.author === user.username ? (
+                <CommentDeleteButton
+                  comment={comment}
+                  commentList={commentList}
+                  setCommentList={setCommentList}
+                />
+              ) : null}
```
Comment (corkesw) [inline src/components/Comments.jsx]: At the moment, if I want to ass a comment, I have to scroll to the bottom of the comment list. Maybe rethink the layout / comment order here

File: src/components/CommentDeleteButton.jsx (hunk 1 above)
Comment (GWorsnop) [reply src/components/CommentDeleteButton.jsx]: Changed the if (err) logic.

File: src/components/Comments.jsx (hunk 2 above)
Comment (GWorsnop) [reply src/components/Comments.jsx]: Changed the add comment section be to outside of the expandable block.  Now shows up when comments do not, also added user feedback if they add a comment.

**Comment by k-huggs** [comment]
all good!

**Pull request merged**